app/src/main/sentences/sl/telephone.dslf

```diff
@@ -0,0 +1,3 @@
+telephone: high
+pokliči|(pozvoni)|telefoniraj|kontaktiraj .who.;
+pokliči|pozvoni .who. ;
```

Comment:
```suggestion
pokliči|pozvoni|telefoniraj|kontaktiraj .who.;
```
You don't need the second sentence anymore, as it's now included in the first one